pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -145,6 +150,39 @@ func (m *managerImpl) IsUnderInodePressure() bool {
 	return hasNodeCondition(m.nodeConditions, api.NodeInodePressure)
 }
 
+func startMemoryThresholdNotifier(thresholds []Threshold, observations signalObservations, hard bool, handler thresholdNotifierHandlerFunc) error {
+	for _, threshold := range thresholds {
+		if threshold.Signal != SignalMemoryAvailable ||
+			(hard && threshold.GracePeriod != time.Duration(0)) ||
+			(!hard && threshold.GracePeriod == time.Duration(0)) {
+			continue
+		}
+		observed, found := observations[SignalMemoryAvailable]
+		if !found {
+			continue
+		}
+		cgroups, err := cm.GetCgroupSubsystems()
+		if err != nil {
+			return err
+		}
+		cgpath, found := cgroups.MountPoints["memory"]
```

Comment:
note that in the future, we plan to support evictions both on root and on `--cgroups-root`. May be its worth adding a TODO here or even better file an issue for it.